src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String NULL_TESTER_STRING = null;
+    private static final Integer NULL_TESTER_INT = null;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void testRegister_ValidUser_Ok() throws RegistrationException {
+        User user = createDefaultUser();
+        User registeredUser = registrationService.register(user);
+        Assertions.assertEquals(user, registeredUser);
+    }
+
+    @Test
+    public void testRegister_UserWithExistingLogin_NotOk() {
+        User user1 = createDefaultUser();
+        User user2 = createDefaultUser();
+        user2.setPassword("password2");
+        Assertions.assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user1);
+            registrationService.register(user2);
+        });
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/49767049/884c0a80-c8cf-494b-a01e-692b2050a4b2)
here, add the first user directly to the storage